tests/ptp/tbc.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2023 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ptp
+
+import (
+	"time"
+	"os"
+	"os/exec"
+	"bufio"
+	"regexp"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-partner-solutions/vse-sync-testsuite/pkg/images"
+	//"github.com/redhat-partner-solutions/vse-sync-testsuite/pkg/pods"
+
+	"github.com/redhat-partner-solutions/vse-sync-testsuite/pkg/clients"
+	"github.com/redhat-partner-solutions/vse-sync-testsuite/pkg/collectors/devices"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	PTP_GROUP_LENGTH = 4
+)
+const (
+	PTP_METRICS = 4
+)
+
+const (
+	ptpLogsProcessed = "/tmp/data.csv"
+)
+
+const (
+	ptpLogs = "/tmp/data.log"
+)
+
+func Readln(r *bufio.Reader) (string, error) {
+  var (isPrefix bool = true
```

Comment:
Convention is `var` blocks are formatted like this:
```go
var (
       isPrefix bool = true
       err error = nil
       line, ln []byte
)
```

Reply:
Done.